pkg/labels/labels.go

```diff
@@ -29,6 +29,12 @@ const (
 	// WARNING: multiple containers may have same the name label
 	Name = Prefix + "name"
 
+	//Project Name
+	Project = "com.docker.compose.project"
```

Comment:
Project -> ComposeProject

Reply:
yep I did the same for Service -> ComposeService